config/adopters.json

```diff
@@ -565,6 +565,15 @@
 				"iot.cyclonedds"
 			]
 		},
+		{
+			"name": "Orange",
+			"homepage_url": "https://www.orange.com/",
+			"logo": "logo-orange.png",
```

Comment:
It would be better if you could provide a svg instead of a png but I guess png is OK too (as other projects use this format)

Reply:
> It would be better if you could provide a svg instead of a png but I guess png is OK too (as other projects use this format)

Yep, svg would be better. 
The format of the logos is depended on the organizations or companies who provide them.